### ISSUE TITLE
TimesNet incompatible weight shape vs. input shape 

### ISSUE THREAD

**Issue by sshen-2004** [thuml/Time-Series-Library#364]
Encounter run time error when using TimesNet model for the short-term forecasting task. Please see the parameters used and error message. Used the custom dataloader.

--task_name
short_term_forecast

--is_training
1

--root_path
./dataset/ETT

--model_id
ett

--model
TimesNet

--data
custom

--features
MS

--e_layers
2

--d_layers
1

--factor
3

--enc_in
1

--dec_in
1

--c_out
1

--batch_size
16

--d_model
64

--des
Exp

--itr
1

--learning_rate
0.001

--loss
SMAPE

_RuntimeError: Given groups=1, weight of size [64, 1, 3], expected input[16, 7, 98] to have 1 channels, but got 7 channels instead._

**Comment by wuhaixu2016** [comment]
Hi, according to the error log, you can change the enc_in, dec_in and c_out as 7.
